packages/video_player/video_player_web/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.8
+
+* Ensure `buffering` state is only removed when the browser reports enough video
```

Comment:
Nit: Ensures

Reply:
@stuartmorgan The idea is not to be able to know if the whole content is buffered but if enough is loaded to ensure a playback with no interruption. So validating that everything from the current position to the end is buffered is not what's needed here.